config/helm/chart/default/templates/Common/customresource-dynakube.yaml

```diff
@@ -106,11 +106,12 @@ spec:
     {{- end }}
 
     {{- if .Values.activeGate.customProperties }}
-      {{- if .Values.activeGate.customProperties.value }}
-    customProperties:
-      value: {{ .Values.activeGate.customProperties.value }}
-      {{- else if .Values.activeGate.customProperties.valueFrom }}
     customProperties:
+      {{- if (.Values.activeGate.customProperties).value }}
```

Comment:
This situation would call for a unit tests, please add one to the Helm unittest suite for the custom resource

Reply:
I didn't check off the unit test entry yet 😉